ui/src/containers/MotorInputContainer.js

```diff
@@ -1,12 +1,12 @@
 import React, { Component } from 'react';
 import { bindActionCreators } from 'redux';
 import { connect } from 'react-redux';
-import * as sampleViewActions from '../actions/sampleview'; // eslint-disable-line import/no-namespace
-import * as generalActions from '../actions/general'; // eslint-disable-line import/no-namespace
-import * as beamlineActions from '../actions/beamline'; // eslint-disable-line import/no-namespace
```

Comment:
Taking the opportunity to remove these namespace imports (note that `generalActions` was not actually used).